src/celeritas/io/ImportProcess.hh

```diff
@@ -80,17 +80,24 @@ enum class ImportProcessClass
 /*!
  * Store physics process data.
  *
- * \note
- * \c ImportPhysicsTable is process and type (lambda, dedx, and so
+ * \note \c ImportPhysicsTable is process and type (lambda, dedx, and so
  * on) dependent, with each table type including physics vectors for all
  * materials. Therefore, the physics vector of a given material is retrieved
  * by finding the appropriate \c table_type in the \c tables vector and
  * selecting the material: \c table.physics_vectors.at(material_id) .
+ *
+ * \todo remove \c secondary_pdg , rename \c particle_pdg to just \c pdg, also
```

Comment:
The reasoning for removing the secondary pdg is because that is already hardcoded in our processes?

Reply:
It's not used anywhere, and even the Geant4 definition is kind of arbitrary ... for example, gamma conversion spits out positrons *and* electrons, but the G4EM model can only contain one particle type.